datadog-service/0.15.0/artifacthub-pkg.yml

```
@@ -0,0 +1,28 @@
+version: 0.15.0
+name: datadog-service
+displayName: datadog-service
+createdAt: '2022-07-12T06:21:51Z'
+description: Keptn SLI provider for datadog
+digest: '2022-07-12T11:58:42Z'
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+- keptn
+- sandbox
+links:
+- name: Source
+  url: https://github.com/keptn-sandbox/datadog-service/tree/release-0.15.0
+annotations:
+  keptn/org: sandbox
```

```suggestion
annotations:
  keptn/kind: observability
  keptn/org: sandbox
  keptn/version: 0.15.x
```

feel free to update keptn/version accordingly ;)

---

Added. Thank you for the suggestion.